sdcflows/workflows/apply/correction.py

```diff
@@ -103,7 +103,9 @@ def init_unwarp_wf(omp_nthreads=1, debug=False, name="unwarp_wf"):
 
     rotime = pe.Node(GetReadoutTime(), name="rotime")
     rotime.interface._always_run = debug
-    resample = pe.Node(ApplyCoeffsField(num_threads=omp_nthreads), name="resample")
+    resample = pe.Node(ApplyCoeffsField(
+        num_threads=omp_nthreads if not debug else 1
```

Comment:
Just as an thought, I wonder if this would be more or less clear:

```Python
    resample = pe.Node(ApplyCoeffsField(num_threads=debug or omp_nthreads), name="resample")
```

Since bools are a subclass of int, it will be 1 if debug is True or omp_nthreads if debug is False.